src/plugins/cachefilter/README.md

```diff
@@ -0,0 +1,16 @@
+- infos = Information about the cachefilter plugin is in keys below
+- infos/author = Marvin Mall <e1225943@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides =
+- infos/recommends =
+- infos/placements =
```

Comment:
Needs some placement in get/set direction

It should be a position late in kdbGet(), but early in kdbSet()